src/Tribe/Tickets.php

```diff
@@ -256,21 +256,21 @@ class Tribe__Tickets__Tickets {
 		 *
 		 * @var string
 		 */
-		protected $key_attendee_first_name = '_tribe_attendee_first_name';
+		const KEY_ATTENDEE_FIRST_NAME = '_tribe_attendee_first_name';
```

Comment:
Lets avoid constants, use `public $key_attendee_first_name`

Reply:
is there a reason why you avoid constants? I find them extremely useful for sharing strings around the codebase, as then you don't have to call an instance of the class just to grab a string.

Reply:
So here is the problem, currently, you cannot deprecate a constant in any capacity or change its name. Which has proven to be a problem in the past.